ci-operator/config/openshift/ibm-roks-toolkit/openshift-ibm-roks-toolkit-release-4.10.yaml

```diff
@@ -0,0 +1,51 @@
+base_images:
+  base:
```

Comment:
Is it possible to use a different base image here? Something like ubi-minimal instead of these CI images that we know have numerous CVEs?

Reply:
Yes, basically we need to mirror the image we need to an imagestream as described here:
https://docs.ci.openshift.org/docs/how-tos/external-images/